CHANGELOG.md

```diff
@@ -1,8 +1,63 @@
-## **5.2.0** <sub><sup>Not released yet</sup></sub>
+## **5.2.0** <sub><sup>2023-10-06 ([238b554...e8fa794](https://github.com/Cosmo-Tech/azure-sample-webapp/compare/238b5547...e8fa794a?diff=split))</sup></sub>
+
+### Features
+
+- improve **user input validation** for scenario parameters:
+  - type-checking
+  - required fields
+  - min/max length or values
+  - list of allowed file extensions
+  - comparison with other scenario parameters (e.g. _start date < end date_)
+  - **error badges** added to point out tabs with validation errors
+- improvements for **table** scenario parameters
+  - add **fullscreen** support for an improved user experience when working with tables
+  - when option `canChangeRowsNumber` is set, users will be able to manually **create and delete rows** in the table
+  - add support for **groups of columns**, with optional columns "collapse & expand"
+- improve checks & error messages to help integrators:
+  - show warnings in console when solution contains unrecognized keys
+  - show warnings in console when workspace contains unrecognized keys
+  - improved error messages context & format for PowerBI service account errors
+- more context shown to end-users:
+  - workspace information is now visible from scenario view
+  - scenario run types are now visible from scenario manager
+  - the technical details of the solution are shown in the "Help" menu
+  - simulation logs can now be downloaded even when scenario runs have succeeded
+- and more new features:
+  - when option `shouldRenameFileOnUpload` is set for files or tables parameters, uploaded files are renamed before being saved
+  - add optional tooltips to describe values of "enum" scenario parameters
+  - add translation support for enum values
+
+### Bug Fixes
+
+- force date values received from \(and sent to\) back\-end to be set to midnight UTC ([987f1cf](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/987f1cfb))
+- rename \.xlsx files imported in Table to \.csv before upload ([5e75dfc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/5e75dfc0))
+- \[PROD\-9531\] prevent aggrid from automatically renaming table columns ([9fcb439](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/9fcb4390))
+- \[PROD\-11247\] make email & roles non\-case\-sensitive ([25ececb](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/25ececbd))
+- \[PROD\-12144\] make running state more visible ([d07c6bc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/d07c6bcd))
+- sligthly increase size of run template name in Scenario view ([1c18af0](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/1c18af02))
+- \[PROD\-7940\] sort scenarios in dropdown list, in scenario creation pop-up ([9d23168](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/9d231682))
```

Comment:
I'd propose to rephrase this line to be more clear about fix:
_add parent-child indentation in scenario dropdown list, in scenario creation pop-up_

The verb _sort_ can be confusing as we are not actually sorting the list but adding visual indentation :slightly_smiling_face:

Reply:
I already rephrased the commit message but your version is actually way better, thanks ! :smile: